src/Caesar.jl

```diff
@@ -29,7 +29,8 @@ using
   CloudGraphs,
   Neo4j,
   Mongo,
-  LibBSON
+  LibBSON,
```

Comment:
Preemptively change to `Mongoc` instead of `Mongo` and `LibBSON` as done on current `master` branch.